src/main/client/src/components/app/table/react-table/ReactTableContainer.js

```diff
@@ -5,12 +5,13 @@ import { setActiveModal } from '../../../../reducers/modal/actions';
 import ReactTable from './ReactTable';
 
 import {
-    toggleSelection, toggleSelectAll, setEditedLayer, setRowFilter, setTableEdited, addFiltered,
+    toggleSelection, toggleSelectAll, setEditedLayer, setRowFilter, setTableEdited, addFiltered, setActivePage
```

Comment:
Keskimmäinen rivi venyy aika pitkäksi. Lisäksi objektimäärittelyissä, jos attribuutteja omalla rivillään, niin rivin perään on tapana lisätä pilkku, kuten commitissa korvatussa rivissä.
```suggestion
    toggleSelection,
    toggleSelectAll,
    setEditedLayer,
    setRowFilter,
    setTableEdited,
    addFiltered,
    setActivePage,
```